app/Http/Controllers/MyGalleryController.php

```diff
@@ -99,7 +99,10 @@ public function edit(Request $request, Gallery $gallery): Response
             'title' => trans('metatags.my_galleries.edit.title', ['name' => $gallery->name]),
             'nfts' => GalleryNftData::collection($nfts),
             'collections' => new GalleryCollectionsData(GalleryCollectionData::collection($collections)),
-            'gallery' => GalleryData::fromModel($gallery, null),
+            'gallery' => GalleryData::fromModel(
+                gallery: $gallery,
+                limit: 16,
```

Comment:
can we move this to a constant so it's easier to update in case we ever change the 16 value to some other? 🤔 same applies to anything else that would reference that 16

Reply:
fyi we do need to add checks for adding 16+ nfts per gallery. Awaiting design: https://app.clickup.com/t/862kc4233

Reply:
https://app.clickup.com/t/861nam5w1 as follow up